node/pkg/governor/mainnet_tokens_test.go

```diff
@@ -44,14 +47,12 @@ func TestTokenListAddressSize(t *testing.T) {
 func TestTokenListChainTokensPresent(t *testing.T) {
 	tokenConfigEntries := tokenList()
 
-	chains := []uint16{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 18}
-
-	/* Assume that all chains will have governed tokens */
-	for _, chain := range chains {
+	/* Assume that all chains within a token bridge will have governed tokens */
+	for chain, _ := range common.KnownTokenbridgeEmitters {
```

Comment:
We can do the same thing here that we did for `TestChainListChainPresent`:

```
	entries := make([]vaa.ChainID, 0, len(tokenConfigEntries))
	for _, e := range tokenConfigEntries {
		entries = append(entries, vaa.ChainID(e.chain))
	}

	emitters := make([]vaa.ChainID, 0, len(common.KnownTokenbridgeEmitters))
	for e := range common.KnownTokenbridgeEmitters {
		emitters = append(emitters, e)
	}

	assert.ElementsMatch(t, entries, emitters)
```

I would make this a suggestion but github in its infinite wisdom doesn't allow comments outside of the diff range 🤦.

Reply:
`tokenConfigEntries` will have duplicate chainIDs in it, so if we compare `entries` (has duplicates) to `emitters` (doesn't have duplicates) it will not match.  This is why I'm keeping the existing strategy on the tokens vs. the chains code where I did make it so.

Reply:
In that case we could use a `map[vaa.ChainID]bool` to deduplicate before comparing but I'm also fine with leaving this as is.

Reply:
Ok, that makes sense, I'll make that change once we settle on the auto-merge PR and I adapt this to include big tx unti-text expectations.